api/src/main/java/org/openmrs/api/cache/CacheConfig.java

```diff
@@ -63,7 +63,7 @@ public class CacheConfig {
 	private String cacheConfig;
 
 	@Bean(name = "apiCacheManager")
-	public SpringEmbeddedCacheManager apiCacheManager() throws IOException {
+	public CacheManager apiCacheManager() throws IOException {
```

Comment:
is this change supposed to be part of the pull request that changes patient and person from hbm to annotations?